src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -194,6 +225,44 @@ public bool VerifyData(ReadOnlySpan<byte> data, ReadOnlySpan<byte> signature, Re
             return VerifyDataCore(data, context, signature);
         }
 
+        /// <summary>
+        ///   Verifies that the specified signature is valid for this key and the provided data.
+        /// </summary>
+        /// <param name="data">
+        ///   The data to verify.
+        /// </param>
+        /// <param name="signature">
+        ///   The signature to verify.
+        /// </param>
+        /// <param name="context">
+        ///   The context value which was provided during signing.
+        ///   The default value is a <see langword="null"/> buffer.
```

Comment:
I think `null` might be a bit confusing given that primitive doesn't distinguish between empty and null